src/envs/behavior.py

```diff
@@ -435,24 +435,26 @@ def make_behavior_option(name: str, types: Sequence[Type], params_space: Box,
     def _policy(state: State, memory: Dict, _objects: Sequence[Object],
                 _params: Array) -> Action:
         assert "has_terminated" in memory
-        assert ("controller" in memory and memory["controller"] is not None
+        assert ("policy_controller" in memory
+                and memory["policy_controller"] is not None
```

Comment:
this assert can be written as `assert memory.get("policy_controller") is not None`